src/styles/tokens/fonts.css

```
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;600;700&display=swap');
+/* 폰트 파일 경로만 public/fonts 로 바꿨다. 나머지 토큰 값은 디자인 시스템 원본 그대로다. */
+@font-face{font-family:'KBFGText';src:url('/fonts/KBFGText-Light.otf') format('opentype');font-weight:300;font-style:normal;font-display:swap}
```

Stylelint (17.14.0)

[error] Expected no quotes around "KBFGText" (font-family-name-quotes)

(font-family-name-quotes)